src/FakeItEasy/ExceptionMessages.cs

```diff
@@ -63,6 +63,12 @@ internal static class ExceptionMessages
         public static string CannotRaiseEventWhenCallingBaseMethod =>
             "The fake cannot raise the event because event subscription calls the base implementation.";
 
+        public static string HandleEventsOnStrictFakes =>
+            @"Strict fakes don't handle calls to event accessors unless configured to do so. You can either:
+- configure the calls explicitly;

```

Comment:
Does "You can either sound like we're recommending they do one of those? Should it be softened to "If you want the fake to handle the call, you can either…"? If you think not, I understand.